data/sheragi/archaeology missions.txt

```diff
@@ -212,19 +212,45 @@ mission "Deep Archaeology 5"
 		conversation
 			`You breathe a sigh of relief as you land safely in the spaceport on <planet>. "Okay, what was that all about?" you ask.`
 			`	"I honestly don't know," says Albert. "I can tell you what I suspect, though. The history books don't actually say when the Deep was first settled. The first documented expedition to the Deep was in the 24th century, and they found several human colonies already established in the region, claiming to date back to the very dawn of space flight. But I think the first settlers were brought there long before that. Brought there on alien ships."`
+			branch "pug"
+				or
+					has "FWC Pug 2: offered"
+					has "FW Pug 3: offered"
 			choice
 				`	"For what purpose?"`
 					goto purpose
 				`	"What aliens? The Quarg?"`
 					goto aliens
+			label pug
+			choice
+				`	"For what purpose?"`
+					goto purpose
+				`	"Do you think it was the Pug?"`
+					goto surprise
 			label purpose
 			`	"I'm not sure," he says. "But I first became interested in the Deep while researching the Alpha Wars. I'm sure you know the history, how humanity was nearly enslaved by psychopathic 'super' humans that we ourselves had created by dabbling in genetic engineering. At that time, the Deep made a sudden, improbable technological leap that allowed them to construct weapons and ships far more powerful than what the Alphas had, and the war against them was won."`
 				goto end
 			label aliens
 			`	"There are popular folk tales in the Deep about 'elves,'" says Albert. "Thin, pale, graceful creatures, with blue blood."`
 			`	John frowns. "Hemocyanic blood. Not the Quarg, then. Their lungs are odd, but their blood uses hemoglobin for oxygen transport, just like ours."`
+				goto end
+			label surprise
+			`	Albert and John look at each other confusedly for a second before Albert says, "The aliens that invaded human space?"`
+			`	John appears to think hard on the subject before saying, "Of course! I don't know how we didn't think of it. From the descriptions that I've heard about them, they almost exactly match the folk tales in the Deep about 'elves.'"`
+			`	"Thin, pale, and blue blooded, yes? Well, I think you may have found our aliens, Captain <last>," replies Albert.`
+			choice
+				`	"That's great, but I think they're far too dangerous to be useful to us now."`
```

Comment:
I don't really get how this choice follows from the previous dialog. The implication with this choice is that Albert and John were planning to contact whatever aliens brought humans to the Deep, yet that wasn't the case. Not really sure what to replace it with, though.

Reply:
That's a good point, seeing as the current state has them excitedly acknowledge this information only to divert it quickly. I suppose they could simply connect the dots and acknowledge it as being interesting, but not relevant, before moving on, though that may make it seem like the player's decision to bring up the Pug was pointless. I'll have to mull over my options for this dialogue for a bit.

Reply:
The latest commit provides a (hopefully) better structured option and dialogue flow.